### PR TITLE
Aladdin generated examples for acr, dls, network, storage, vm

### PR DESCRIPTION
**Pull request by AladdinBI**
- Sort commands alphabetically in help files for easy diff.
 - Add Aladdin crafted examples.

### CODE REVIEW

File: src/command_modules/azure-cli-dls/azure/cli/command_modules/dls/_help.py [hunk 1]
```diff
@@ -152,6 +164,10 @@
 helps['dls account update'] = """
 type: command
 short-summary: Updates a Data Lake Store account.
+examples:
+  - name: Updates a Data Lake Store account. (autogenerated)
+    text: az dls account update --account mydatalakestoragegen1 --allow-azure-ips Enabled --firewall-state Enabled --resource-group MyResourceGroup --subscription MySubscription --tags key[=value]
```
Comment (tjprescott) [inline src/command_modules/azure-cli-dls/azure/cli/command_modules/dls/_help.py]: The value for tags here is a usage string, not an example.

File: src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py [hunk 2]
```diff
@@ -63,6 +63,9 @@
     text: |
         end=`date -v+30M '+%Y-%m-%dT%H:%MZ'`
         az storage account generate-sas --permissions cdlruwap --account-name MyStorageAccount --services qt --resource-types sco --expiry $end -otsv
+  - name: Generates a shared access signature for the account (autogenerated)
+    text: az storage account generate-sas --account-key 00000000 --account-name $my_diagnostic_storage_account --expiry $end -otsv --https-only --permissions acuw --resource-types co --services bfqt
```
Comment (tjprescott) [inline src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py]: `$my_diagnostic_storage_account` is bash-specific syntax. Why not simply use the normal "myStorageAccount" name convention?

File: src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py [hunk 3]
```diff
@@ -191,6 +200,9 @@
   - name: Copies a blob asynchronously. Use `az storage blob show` to check the status of the blobs. (autogenerated)
     text: az storage blob copy start --account-key 00000000 --account-name MyAccount --destination-blob MyDestinationBlob --destination-container MyDestinationContainer --source-uri https://storage.blob.core.windows.net/photos
     crafted: true
+  - name: Copies a blob asynchronously. Use `az storage blob show` to check the status of the blobs (autogenerated)
+    text: az storage blob copy start --account-name MyAccount --destination-blob MyDestinationBlob --destination-container MyDestinationContainer --sas-token $sas --source-uri https://storage.blob.core.windows.net/photos
```
Comment (tjprescott) [inline src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py]: `$sas` is shell-specific

File: src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py [hunk 4]
```diff
@@ -313,6 +325,9 @@
     text: |
         end=`date -d "30 minutes" '+%Y-%m-%dT%H:%MZ'`
         az storage blob generate-sas --account-name MyStorageAccount -c MyContainer -n MyBlob --permissions r --expiry $end --https-only
+  - name: Generates a shared access signature for the blob. (autogenerated)
+    text: az storage blob generate-sas --account-key 00000000 --account-name MyStorageAccount --container-name MyContainer --expiry $end --name MyBlob --permissions r
```
Comment (tjprescott) [inline src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py]: `$end` is shell specific

File: src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py [hunk 5]
```diff
@@ -986,6 +1034,9 @@
         end=`date -d "30 minutes" '+%Y-%m-%dT%H:%MZ'`
         sas=`az storage share generate-sas -n MyShare --account-name MyStorageAccount --https-only --permissions dlrw --expiry $end -otsv`
         az storage file upload -s MyShare --account-name MyStorageAccount --source file.txt  --sas-token $sas
+  - name: Generates a shared access signature for the share. (autogenerated)
+    text: az storage share generate-sas --connection-string $connectionString --expiry 2019-02-01T12:20Z --name MyShare --permissions dlrw
```
Comment (tjprescott) [inline src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py]: `$connectionString` shell-specific

**Comment by albaloo** [comment]
@tjprescott Thanks for the comments. We addressed all your comments, but because of the way our pipeline works, we had to send a new PR to address your comments. I will close this one.

**Pull request closed without merging**